hazelcast/src/main/java/com/hazelcast/map/impl/query/QueryResultCollection.java

```diff
@@ -44,6 +44,11 @@ public QueryResultCollection(SerializationService serializationService, Iteratio
         }
     }
 
+    // just for testing
+    public Collection<QueryResultRow> getRows() {
```

Comment:
yeah, I appreciate it's not a biggie; but still no reason to expose more than is necessary.